### ISSUE TITLE
Crash when select some layers

### ISSUE THREAD

**Issue by salexpro** [avocode/avocode#2202]
---
* **Version**: 3.6.0
* **Sketch plugin version**: 51.2.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 30893
---
```
TypeError: Cannot read property 'set' of undefined
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052319
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1973:22)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at Map.updateIn (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1280:26)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052283
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3907:58
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2701:43
    at List.__iterate (/
```

**Comment by vojtatranta** [comment]
Hi @salexpro,
does it happen when you select any layer? Or does it happen just on one specific design?

**Comment by salexpro** [comment]
It happens when I select any layer of Figma component. Besides this components is not in components tab. When I try to click on it, avocode says:
![image](https://user-images.githubusercontent.com/4237964/47300021-30299b80-d624-11e8-8b14-c9cfb3a7f4bf.png)

![image](https://user-images.githubusercontent.com/4237964/47299718-8cd88680-d623-11e8-9e2f-34b3c15ca140.png)

**Comment by crashmajstr** [comment]
Hi @salexpro , we're currently working on a fix, I am sorry for the inconvenience, we'll let you know once it's fixed. 
Best,
Josef

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ll release the fix in an upcoming hour or so. It’ll be version 3.6.1.
For now, you can just download and use our older stable build 3.5.6 

Download links
Mac: https://we.tl/t-bS4jyZufpT
Win: https://we.tl/t-12nDKmpafd
Linux: https://we.tl/t-krrbw7ZRvB

I’ll let you know once the fix is out. Sorry for the inconvenience.

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 containing the fix for this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Hope everything will be ok from now on.
I am very sorry for the inconvenience. 
On behalf of Avocode,
Josef Kettner

**Comment by salexpro** [comment]
Still, receive the same error
Do I need to re-upload my Figma design?
![image](https://user-images.githubusercontent.com/4237964/47361146-e9967880-d6d9-11e8-898b-b4d107118db0.png)
![image](https://user-images.githubusercontent.com/4237964/47361173-f9ae5800-d6d9-11e8-8f30-0863225f74e4.png)


---
* **Version**: 3.6.1
* **Sketch plugin version**: 51.2.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 30893
---
```
TypeError: Cannot read property 'set' of undefined
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052349
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1973:22)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at Map.updateIn (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1280:26)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052313
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3907:58
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2701:43
    at List.__iterate (/
```

**Comment by jakubzitny** [comment]
Could you please send us the private/public to the design where this is happening, please @salexpro?

**Comment by vojtatranta** [comment]
@salexpro this is definitely fixed in `3.6.2`

**Comment by crashmajstr** [comment]
@salexpro already replied via intercom that it's fixed and working ok. cc @vojtatranta @jakubzitny
